transport/tcp/serve_tcp_ipmap.go

```diff
@@ -60,38 +46,62 @@ func (m *ipMap) Get(ipAddr *net.IPAddr) (string, error) {
 			return e.ident, nil
 		}
 		if e.subnet != nil {
-			contains := e.subnet.Contains(ipAddr.IP)
-			if contains {
-				if e.ident == "willbeclientip" {
-					return ipAddr.IP.String(), nil
-				}
-				return e.ident + ipAddr.IP.String(), nil
+			if e.subnet.Contains(ipAddr.IP) {
```

Comment:
take advantage of short-circuiting by making the encloding if-condition `e.subnet != nil && e.subnet.Contains(ipAddr.IP)` => makes the code more concise